src/utils/image.js

```diff
@@ -219,30 +106,28 @@ export async function deleteImage(hash) {
 	const settings = await getSettings();
 	const host = await getHostSettings();
 
-	if (host === aHosts[1]) {
-		await doRequest(
-			hash,
-			'GET',
-			null,
-			null,
-			() => {},
-			() => {},
-			() => {}
-		);
-	} else {
-		await doRequest(
-			`https://api.imgur.com/3/image/${hash}`,
-			'DELETE',
-			null,
-			{
-				text: 'Authorization',
-				value: `Client-ID ${settings.apiClientId}`
-			},
-			() => {},
-			() => {},
-			() => {}
-		);
-	}
+	const get = host === aHosts[1];
+
+	await doRequest(
+		get ? hash : `https://api.imgur.com/3/image/${hash}`,
+		get ? 'GET' : 'DELETE',
+		null,
+		get
+			? null
+			: {
+					text: 'Authorization',
```

Comment:
Is prettier on drugs?

Reply:
it's the nested ternaries with the object there, so it ends up looking like that